### DISCUSSION TITLE
User-defined help menu bot

### DISCUSSION THREAD

**Discussion by chetanyakan** [mattermost/community#107] [General Ideas]
A bot that posts previously configurable messages to users.
This can be useful to teach new users etiquette for using Mattermost.

UX can be similar to the Reactiflux-discord bot:
![image](https://user-images.githubusercontent.com/35728906/123304458-99cc7700-d53c-11eb-95cc-7663f3cb5a99.png)
![image](https://user-images.githubusercontent.com/35728906/123304512-a51fa280-d53c-11eb-90d3-244e18fe03db.png)
![image](https://user-images.githubusercontent.com/35728906/123304535-abae1a00-d53c-11eb-939a-d5a050cb033e.png)
![image](https://user-images.githubusercontent.com/35728906/123304572-b49eeb80-d53c-11eb-93aa-f088a9fa3925.png)
![image](https://user-images.githubusercontent.com/35728906/123304601-bec0ea00-d53c-11eb-84a6-5e46c5f0fb17.png)

**Comment by esethna** [comment]
Ya we have "Welcome Bot" but we can certainly do better when it comes to onboarding new users
